scripts/run-configuration.sh

```diff
@@ -4,6 +4,7 @@ set -Eeuxo pipefail
 
 cd "$( dirname "${BASH_SOURCE[0]}" )/.."
 
+# shellcheck disable=SC2006
```

Comment:
I set it to disable because it is a backtick that is written in markdown.

https://github.com/koalaman/shellcheck/wiki/SC2006